index.html

```diff
@@ -20,7 +20,7 @@
   <header class="main-header flex pd-tp-bt">
     <section class="welcome">
       <a href="./index.html">
-        <h3>Welcome</h3>
+        <h1>Welcome</h1>
```

Comment:
As mentioned in my first inline comment, kindly state the lines of code where you've made these changes in the evaluation report of your PR :blush: 
Kindly do this for all affected headings. I suggest this should be a single line (all changes related to headlines). You can then separate the lines of code with a comma.

Reply:
Resolved this